### PR TITLE
add nathan glaser picture and bio to website 

### PR DESCRIPTION
**Pull request by nglaser3**
## Summary of changes

Adding my picture and bio to people page of website.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests.md) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

**Review by smpark7 — CHANGES_REQUESTED** [review]
This looks mostly good. You need to fix the format as I've suggested below. I'll make a separate PR to fix Ceser's formatting.

File: _data/people.yml [hunk 1]
```diff
@@ -159,6 +159,20 @@
       icon: github-alt
       name: Github
 
+-name: Nathan Glaser
+ pic: /img/people/glasern.png
+ dates: 2023-Present
+ role: Undergraduate Research Assistant
+ description: 'Nathan is an undergraduate in NPRE at UIUC, interested in fission reactor fuel modeling and simulation.'
+ social:
```
[the remark this answers is not in the document]
Comment (nglaser3) [reply _data/people.yml]: Ok, thank you for catching that, it has been fixed

File: _data/people.yml [hunk 2]
```diff
@@ -173,3 +187,5 @@
     - url: https://www.google.com/maps/d/edit?mid=zwJ5Sc8HIDN4.k3fYHsg-biWE&usp=sharing
       icon: map-marker
       name: 118 Talbot Lab
+
+
```
[the remark this answers is not in the document]
Comment (nglaser3) [reply _data/people.yml]: Thanks for the suggestion, this has been resolved

**Comment by nglaser3** [comment]
@smpark7 Thank you for your review, please let me know if there is anything else for me to fix!

**Review by smpark7 — APPROVED** [review]
Looks good to me! Good job with the fixes!

**Pull request merged**